lib/spack/docs/build_systems/intelpackage.rst

```diff
@@ -558,43 +558,29 @@ follow `the next section <intel-install-libs_>`_ instead.
              modules:    []
              spec:       intel@18.0.3
              paths:
-               cc:       stub
-               cxx:      stub
-               f77:      stub
-               fc:       stub
+               cc:       /usr/bin/true
+               cxx:      /usr/bin/true
+               f77:      /usr/bin/true
+               fc:       /usr/bin/true
 
       Replace ``18.0.3`` with the version that you determined in the preceding
       step. The contents under ``paths:`` do not matter yet.
```

Comment:
I think it would be helpful to note here that they must exist (although you clarify that later too).

Reply:
Ok, now directly mentioned underneath the code snippet.